guides/source/form_helpers.md

```diff
@@ -404,22 +419,24 @@ Rails works around this issue by emulating other methods over POST through a com
 </form>
 ```
 
+In this case, the "Update" button will be treated as `PATCH` and the "Delete" button will be treated as `DELETE`.
+
 [formmethod]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attr-formmethod
 [button-name]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attr-name
 [button-value]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attr-value
 
 Making Select Boxes with Ease
```

Comment:
(opitional, nitpick): This title feels a lengthy, I lead towards something simple like "Select Boxes" or even "Drop-Downs.

Reply:
All of the previous title start with -ing word, leaving it as is for consistency.